streaming-http-channel-2023.html

```diff
@@ -0,0 +1,776 @@
+In the context of a Web Browser, the Streaming HTTP Channel provides a convenient mechanism for a Solid Storage
+to alert a subscribing client of these changes.</p>
+   <p>This document describes a Solid Notifications channel type that makes use of the Fetch API.</p>
+   <p>This specification is for:</p>
+   <ul>
+    <li data-md>
+     <p>Resource server developers who wish to enable clients to listen for updates to particular resources.</p>
+    <li data-md>
+     <p>Application developers who wish to implement a client to listen for updates to particular resources.</p>
+   </ul>
+   <h3 class="heading settled" data-level="1.1" id="terminology"><span class="secno">1.1. </span><span class="content">Terminology</span><a class="self-link" href="#terminology"></a></h3>
+   <p><strong>This section is non-normative.</strong></p>
+   <p>This document uses terminology from the Solid Notification Protocol, including "subscription API", "gateway API".
+This document also uses terms from The OAuth 2.0 Authorization Framework specification,
+including "resource server", "authorization server", "access token", and "client",
+as well as terms from the WebSub specification, including "topic".</p>
+   <h2 class="heading settled" data-level="2" id="channel-type"><span class="secno">2. </span><span class="content">StreamingHTTPChannel2023 Type</span><a class="self-link" href="#channel-type"></a></h2>
+   <p>This specification defines the StreamingHTTPChannel2023 type for use with Solid Notifications channels.
+that use the Fetch API.</p>
+   <p>An StreamingHTTPChannel2023 MUST conform to the <a href="https://solid.github.io/notifications/protocol#discovery">Solid Notifications Protocol</a>.</p>
+   <p>An StreamingHTTPChannel2023 SHOULD support the <a href="https://solid.github.io/notifications/protocol#notification-features">Solid Notifications Features</a>.</p>
+   <p>The StreamingHTTPChannel2023 type further constrains following properties properties:</p>
+   <dl>
+    <dt data-md>receiveFrom
+    <dd data-md>
+     <p>The <a href="https://solid.github.io/notifications/protocol#notify-receiveFrom">receiveFrom</a> property
+is used in the body of the subscription response.
+The value of source property MUST be a URI, using the <code>https</code> scheme.</p>
+   </dl>
+   <p>A client establishes a subscription using the StreamingHTTPChannel2023 type by sending an authenticated
+subscription request to the Subscription Resource retrieved via Solid Notifications discovery.</p>
+   <p>For StreamingHTTPChannel2023 interactions, the client sends a JSON-LD payload to the appropriate
+Subscription Resource via POST. The only required fields in this interaction are type and topic.
+The type field MUST contain the type of channel being requested: <code>StreamingHTTPChannel2023</code> The topic field MUST contain the URL of the resource a client wishes to subscribe to changes.</p>
```

Comment:
```suggestion
   <p>A client establishes a subscription using the <code>StreamingHTTPChannel2023</code> type by sending an authenticated
subscription request to the Subscription Resource retrieved via Solid Notifications discovery.</p>
   <p>For <code>StreamingHTTPChannel2023</code> interactions, the client sends a JSON-LD payload to the appropriate
Subscription Resource via <code>POST</code>. The only required fields in this interaction are <code>type</code> and <code>topic</code>.
The <code>type</code> field MUST contain the type of channel being requested: <code>StreamingHTTPChannel2023</code>. The <code>topic</code> field MUST contain the URL of the resource to which changes a client wishes to subscribe.</p>
```

Reply:
I'm sorry, this file doesn't really belong here and it is already gitignored on `main`. I'm going to adapt this suggestion to the source which is already in the repo